src/Kanbas/Courses/Quizzes/Editor/index.css

```diff
@@ -0,0 +1,7 @@
+.wd-tabs .nav-item {
```

Comment:
Make the tabs component a common component so that it can be reused.

Reply:
This way "wd-tabs" naming makes sense.

Reply:
Iff this is a bootstrap component, Moving the CSS classes to a common stylesheet is fine. Don't add it to the global one.